compiler/src/iree/compiler/Codegen/Dialect/Codegen/IR/IREECodegenAttrs.td

```diff
@@ -28,24 +28,26 @@ def CPU_DataTiling
 
 def LLVMGPU_Default
     : I32EnumAttrCase<"LLVMGPUDefault", 100>;
+def LLVMGPU_BaseLowering
+    : I32EnumAttrCase<"LLVMGPUBaseLowering", 101>;
```

Comment:
Add this as a new value at the bottom (110) rather than incrementing all of the other pipeline enums.

Reply:
This is fine. We don't need to preserve the numeric value of the passes anyway. This actually gives us a more logical order.